src/app/core/dto/user.ts

```diff
@@ -20,7 +20,8 @@
 export interface User {
 
   id?: string;
-  username: string; // email
+  name: string;
+  email: string;
   groups: string[];
```

Comment:
The structure of the data returned by the back-end will be slightly different. See [engine PR 125](https://github.com/Lumeer/engine/pull/125). However, you can map that to the model structure which best fits your usage needs.